orchestrator/src/main/java/com/scottlogic/deg/orchestrator/generate/GenerateCommandLine.java

```diff
@@ -40,9 +44,17 @@ public void run() {
         task.run();
     }
 
-    @CommandLine.Parameters(index = "0", description = "The path of the profile json file.")
+    @CommandLine.Option(
+        names = {"-p", "--profile-file"},
+        required = true,
+        description = "The path of the profile json file.")
     File profileFile;
 
+    @CommandLine.Option(
+        names = {"-op", "--output-path"}, order = 0,
```

Comment:
The Unix instinct in me says that we shouldn't use `-op` as an option, because the Unix convention is for `-op` to be equivalent to `-o -p`

Reply:
updated